package.json

```diff
@@ -67,6 +67,7 @@
     "grunt-contrib-clean": "^0.6.0",
     "grunt-contrib-watch": "^0.6.1",
     "grunt-simple-mocha": "^0.4.0",
+    "haikro": "^2.3.2",
```

Comment:
I found out it only works with Bash, had me running in circles wondering why my shell wouldn't work. Opening an issue for this :-)